plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/totals/footer-item/index.tsx

```diff
@@ -39,6 +39,7 @@ export interface TotalsFooterItemProps {
 	 */
 	values: LooselyMustHave< CartResponseTotals, 'total_price' | 'total_tax' >;
 	className?: string;
+	isCart?: boolean;
```

Comment:
I'm thinking we could rename this `isCart` for better clarity  to `isEstimatedTotal` or `isSubtotal` if we change it. Or we could say `isCompleteTotal` or `isFinalizedTotal`

Reply:
`isEstimate`? I think that reflects what we're showing rather than "where" we're showing it.